src/publisher/management/commands/ingest.py

```diff
@@ -7,19 +7,20 @@
 
 """
 from collections import OrderedDict
```

Comment:
a feedback from Digirati's and our containerized usage has been: sooner or later we could try to make the ingest called by bot-lax-adaptor available through HTTP rather (or in substitution) of them relying on being on the same filesystem and creating a new Lax process. This would decouple them and let them run in different containers. We don't want to do decisions here that makes this difficult, at the moment this command seems self-contained and could be restructured into an API if needed. :+1:

Reply:
we'll cross that bridge when we come to it.